lib/fedex/request/tracking_information.rb

```diff
@@ -63,8 +68,9 @@ def build_xml
         builder.doc.root.to_xml
       end
 
+      # Use version 6 of tracking API
       def service
-        { :id => 'trck', :version => 5 }
+        { :id => 'trck', :version => 6 }
```

Comment:
I don't really understand what this mean, but sounds like it could be a breaking change for someone trying to upgrade the API. If that's the case it may be harder to sell to the repo owners than adding a way to configure which tracking API version to use